include/model.php

```diff
@@ -335,7 +219,7 @@ public function terms_clauses( $clauses, $lang ) {
 	 * @param bool $filter True if we should return only valid registered post types.
 	 * @return string[] Post type names for which Polylang manages languages and translations.
 	 */
-	public function get_translated_post_types( $filter = true ) {
+	public function get_translated_post_types( bool $filter = true ): array {
```

Comment:
I would not type the param.

Reply:
Reverted in [1b6f7ed](https://github.com/polylang/polylang/pull/1503/commits/1b6f7ed2f498c60d999730b027609e01bfcffd28).